tests/standardise/test_standardise.py

```diff
@@ -656,3 +656,66 @@ def test_standardise_surface_niwa(caplog):
     assert "LAU" in results[0]["site"]
     assert "10m" in results[0]["inlet"]
     assert "niwa" in results[0]["source_format"]
+
+
+def test_standardise_footprints_satellite_raises_error():
+    """
+    Tests standardise footprint raises value error when site and obs_region values are not supplied.
+    """
+    datapath = get_footprint_datapath("GOSAT-BRAZIL-column_SOUTHAMERICA_201004_compressed.nc")
+
+    satellite = "GOSAT"
+    network = "GOSAT"
+    domain = "SOUTHAMERICA"
+
+    with pytest.raises(ValueError):
+        standardise_footprint(
+            filepath=datapath,
+            satellite=satellite,
+            network=network,
+            model="CAMS",
+            height=6500,
```

Comment:
Even for the fail test, for consistency this should use `inlet` key word and be set to "column"
```suggestion
            inlet="column",
```

Reply:
oh nice catch